src/ansiblelint/rules/name.py

```diff
@@ -69,7 +69,10 @@ def matchtask(
     def _check_name(
         self, name: str, lintable: Lintable | None, linenumber: int
     ) -> MatchError | None:
-        if not name[0].isupper():
+        # This rules applies only to scripts that do have uppercase and
+        # lowercase letter, so we ignore anything else. On Unicode islower()
+        # is not the opposite of islower()
```

Comment:
```suggestion
        # This rules applies only to languages that do have uppercase and
        # lowercase letter, so we ignore anything else. On Unicode isupper()
        # is not necessarily the opposite of islower()
```

Reply:
Thanks, I included the fix into https://github.com/ansible/ansible-lint/pull/2351